include/z64actor.h

```diff
@@ -512,6 +512,46 @@ typedef enum {
     /* 0xFF */ NAVI_ENEMY_NONE = 0xFF
 } NaviEnemy;
 
+// A set of 4 triangles which appear around an actor when the player Z-Targets it
+typedef struct {
+    /* 0x00 */ Vec3f pos;
+    /* 0x0C */ f32 radius; // distance towards the center of the locked on
+    /* 0x10 */ Color_RGB8 color;
+} LockOnTriangleSet; // size = 0x14
+
+typedef struct {
+    /* 0x00 */ Vec3f fairyPos; // Used by Navi to indicate a targetable actor or general hint
```

Comment:
Maybe an unpopular opinion, but I like specifically saying navi for all of these "fairy" things. I know the motivation is sync between games, but they are special fairies, separate from normal ones. 
They deserve singling out in the case of both games, in my opinion. Would personally support mm using "tatl" as well.

Reply:
Yeah makes sense, I was thinking about that too as I was documenting this. I was split down the middle between fairy or navi, so I've gone and changed it